app/assets/javascripts/misc/previous-address.js

```diff
@@ -0,0 +1,37 @@
+import 'classlist.js';
+
+const I18n = window.LoginGov.I18n;
+
+function previousAddress() {
+  const accordion = document.querySelector('.accordion');
+
+  if (accordion) {
+    const controls = accordion.querySelector('[aria-controls]');
+    const selects = accordion.querySelectorAll('select');
+    const inputs = accordion.querySelectorAll('input');
+
+    controls.classList.remove('display-none');
```

Comment:
This is fine, I think. The other way is to add a `.no-js` class in the css file and hide elements nested under it.